### DISCUSSION TITLE
Would like the Thunderbird Send button to send immediately.  I would use Send Later otherwise.

### DISCUSSION THREAD

**Discussion by McVette** [Extended-Thunder/send-later#345] [Ideas]
The built-in Send button results in executing Send Later after the default delay.  I'd like the option of Send being immediate and use of the Send Later button to, well, Send Later.  Yes, there is the ability to send immediately once in the Send Later dialog, but that is two steps instead of one.

**Comment by jph-sendlater** [comment]
If I'm understanding you correctly, this is something that can be set in the extension preferences. If you have "Send Does Send Later" unchecked, then the Send button should just send immediately. This should be the default behavior.
